app/controllers/users_controller.rb

```diff
@@ -3,11 +3,24 @@ def new
     @user = User.new
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to @user
```

Comment:
You forgot to fix here too.
You should redirect to `edit_user_path` if user was saved